src/app/embed/embed.component.ts

```diff
@@ -64,4 +70,20 @@ export class EmbedComponent implements OnInit {
     this.artworkUrl = this.artworkUrl || this.feedArtworkUrl;
   }
 
+  @HostListener("window:resize", [])
+  setEmbedHeight() {
+    if (window.parent && window.parent.postMessage) {
+      window.parent.postMessage(JSON.stringify({
+        src: window.location.toString(),
+        context: 'iframe.resize',
+        height: 185
```

Comment:
we can try e.g. measuring the height of the content internally, though I can't say I love that for now since different browsers may behave differently on that front. Maybe a TODO comment?